spec/javascripts/app/utils/events_spec.js

```diff
@@ -0,0 +1,89 @@
+const Events = require('../../../../app/assets/javascripts/app/utils/events').default;
+/*eslint-disable */
+const sinon = require('sinon');
```

Comment:
Sinon looks cool, do you think we should start moving in this direction for javascript tests? I tend to rely on feature specs but I know that's probably not the right way if we continue to add more javascript.